### PR TITLE
800 801 minor demo changes and bug fixes

### LINKED ISSUE #800: Demo App create process model hangs in loading state
### Current Behavior
on the demo application
 - attempt to create a new process
   - application hangs in a loading state in the model (though you can see the process is successfully created if you refresh)
 - the process returned has `isVirtual === false` (this is a hambda ticket: ospin-web-dev/hambda#490)
---

### Intended Behavior/Acceptance Criteria
 - The create process model does not hang

### LINKED ISSUE #801: Minor Demo UI Change Requests
Several small UI requests from Nina. They are small so bundling them together here:

### Feature Description / Problem Solved by Feature
- "Maybe the contact us button should be OSPIN red instead of blue? I am a bit skeptical about the blue-green combination...Or maybe you can use the OSPIN blue (reactor color when process is running) for both buttons..."

- “This is a demo version of the Ospin web application that will give you a taste of our state of the art Modular Bioprocessing technology.” -> Could you write OSPIN since it is an acronym? Also, maybe something like “an overview of our Modular Bioprocessing technology state of the art?” It is just that the sentence you wrote sounds incomplete to me (no “connection” between “art” and “MB technology”), but well you know obviously English better than me :smile: so it might be crap what I am saying."

### Deliverables
...on the splash page
- change the 'contact us' button color to be OSPIN red, making it the same fill type as the 'Try Demo' button with the same on hover effect (so red 'Contact Us!' button, with white font). 
- 'Contact us!' -> 'Contact Us!' (to match the casing in 'Try Demo'
- Make text fonts the same size for both buttons
- Update text per request above

### PR DESCRIPTION
**Pull request by DanielSeehausen**
first commit closes #800 
following commits closes #801

### CODE REVIEW

File: src/components/device/modals/CreateProcessModal.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
 import { addPhase } from 'utils/process'
-import { loadProcessViewer } from 'redux/actions/actions'
+import { loadProcessViewer, addProcessToProcessesList } from 'redux/actions/actions'
```
Comment (ConorTighe1995) [inline src/components/device/modals/CreateProcessModal.jsx:5]: Nice catch, I spotted this error earlier and added it to one of my prs.

File: src/components/authentication/Login.jsx [hunk 2]
```diff
@@ -93,12 +92,13 @@ class Login extends React.Component {
                   </Button>
                   <Divider />
                   <Button
-                    inverted
-                    color='blue'
+                    size='large'
+                    positive
```
Comment (ConorTighe1995) [inline src/components/authentication/Login.jsx:96]: I think I didnt use positive here because of the shade of blue or something.

Comment (DanielSeehausen) [reply src/components/authentication/Login.jsx:96]: I will double check this now!

**Comment by DanielSeehausen** [comment]
this just flickered

**Pull request merged**